src/util/tracker/gameCommands.ts

```diff
@@ -0,0 +1,181 @@
+	const playtimePer = `day: ${makeTimeString(
+		Math.round(totalPlaytime / (range / (86400 * 1000)))
+	)}\nweek: ${makeTimeString(Math.round(totalPlaytime / (range / 604800000)))}\nmonth: ${makeTimeString(
+		Math.round(totalPlaytime / (range / 2628000000))
+	)}\nuser: ${makeTimeString(Math.round(totalPlaytime / users))}\nlog: ${makeTimeString(
+		Math.round(totalPlaytime / totalLogs)
+	)}`;
+	// temporary sorted list of most playtime/log (users)
+	const tmp = db.users.sort((a, b) => b.playtime / b.logs - a.playtime / a.logs)[0];
+	// make most playtime/log string
+	const mostPlaytime = `<@${tmp.id}>: ${makeTimeString(tmp.playtime / tmp.logs)} - ${
+		tmp.logs
+	} logs\nTotal playtime: ${makeTimeString(tmp.playtime)}`;
+
+	const embed = new EmbedBuilder()
+		.setAuthor({
+			name: config.botName,
+			url: config.githubURL,
+			iconURL: config.iconURL,
+		})
+		.setColor(config.color)
+		.setTitle(`Tracking stats about ${targetGame}`)
+		.addFields(
+			{ inline: true, name: "Most logged users", value: mostLogged },
+			{ inline: true, name: "User with most playtime", value: mostPlayed },
+			{ inline: false, name: "_ _", value: "_ _" },
+			{ inline: true, name: "Latest logs", value: latestLogs },
+			{ inline: true, name: "(Average) playtime per", value: playtimePer },
+			{ inline: false, name: "_ _", value: "_ _" },
+			{ inline: true, name: "(Average) most playtime/log", value: mostPlaytime },
+			{ inline: false, name: "_ _", value: "_ _" },
+			{
+				inline: true,
+				name: "Record range",
+				value: `${discordTimestamp(Math.floor(firstSeen / 1000))} -> ${discordTimestamp(
+					Math.floor(Date.now() / 1000)
+				)}(now)\n${makeTimeString(Date.now() - firstSeen)}`,
+			},
+			{ inline: false, name: "_ _", value: "_ _" },
+			{ inline: true, name: "Total logs", value: totalLogs.toString() },
+			{ inline: true, name: "Total playtime", value: makeTimeString(totalPlaytime) }
+		);
+
+	await interaction.reply({ embeds: [embed] });
+}
+export async function gameLast(interaction: ChatInputCommandInteraction) {
+	// get the target game
+	const targetGame = interaction.options.getString("game", true);
+	// load games db
+	const db = trackerGames.get(targetGame);
+	if (!db) {
+		await interaction.reply(GAMENOENTRY);
+		return;
+	}
+
+	// get latest logs
+	const logs = db.lastlogs.reverse();
+	// store future embed fields
+	const fields: APIEmbedField[] = [];
+
+	// make embed field for every log
+	await logs.forEach(async (log) => {
+		// get and validate log (db.lastlogs is a list of numbers as strings, not the actual log)
+		const entry = trackerLogs.get(log);
+		if (!entry) return;
+		// get user who owns the log
+		const user = await interaction.client.users.fetch(entry.userid);
+		// skip if user doesnt exist anymore
+		if (!user) return;
+
+		// make field
+		fields.push({
+			inline: true,
+			name: user.username,
+			value: `<t:${Math.floor(new Date(entry.time).getTime() / 1000)}:d><t:${Math.floor(
```

Comment:
There are functions for that. And please use DateTime for consistency

Reply:
This way its easier to make mistakes and with functions it looks better

Reply:
Of course there is a way. Just use the discordTimestamp method... It does everything you need. You don't need to reinvent the wheel. Also use serializers and deserializers for the date!!!

Reply:
I added Timestamps for every Timestamp format discord provides and added one which combines Short Date and Short Time.
And OMG! Thanks for pointing me so serializers!!! So much stuff is way cleaner now :D